samples/drivers/display/src/main.c

```diff
@@ -50,16 +51,19 @@ static void fill_buffer_argb8888(enum corner corner, uint8_t grey, uint8_t *buf,
 
 	switch (corner) {
 	case TOP_LEFT:
-		color = 0x00FF0000u;
+		color = 0xFFFF0000u;
 		break;
 	case TOP_RIGHT:
-		color = 0x0000FF00u;
+		color = 0xFF00FF00u;
 		break;
 	case BOTTOM_RIGHT:
-		color = 0x000000FFu;
+		color = 0xFF0000FFu;
 		break;
 	case BOTTOM_LEFT:
-		color = grey << 16 | grey << 8 | grey;
+		color = 0xFF000000u | grey << 16 | grey << 8 | grey;
+		break;
+	case CENTER:
+		color = 0x00FFFFFFu | grey << 24;
```

Comment:
I think it would be more intuitive for the background to be transparent (`bg_color = 0x00u;`) and the center rectangle being white?

```suggestion
		color = 0xFFFFFFFFu | grey << 24;
```

Reply:
Sure, we can do that, then I should remove the `grey` fading of the transparency though.

The patch currently fades from transparent to white.

Reply:
Fixed

Reply:
![image](https://github.com/user-attachments/assets/ce1c5287-7556-411a-9f44-baf8d831b01d)